### ISSUE TITLE
How to extract virus sequence

### ISSUE THREAD

**Issue by mengyuan-JI** [jessieren/DeepVirFinder#49]
Hi, I tried using DVF to identify viral sequences from my metagenomic dataset, but how do I extract the identified sequences? There is only a txt file in the output file.
Thanks a lot

**Comment by YoungerRen** [comment]
i think u can extract ur virus sequence according to the blog --https://www.jianshu.com/p/7873ad67620f
 How big is your metagenomic dataset dataset? How long has it been running with deepvirfinder, my input file is 1G and it has been running for 3 days with 96 threads, is this normal?

**Comment by mengyuan-JI** [comment]
thank you~ I will try and let you know, but for me 3 days with 96 threads sounds very long.

**Comment by YoungerRen** [comment]
it's my first time to run the DVF, so i do not know the average running time. however, It's torture for me to spend more than three days running a program with no results.

**Comment by yqy6611** [comment]
You can use seqkit

**Comment by Sinceter** [comment]
> it's my first time to run the DVF, so i do not know the average running time. however, It's torture for me to spend more than three days running a program with no results.

@YoungerRen Yes, for my input of ~200 MB, it takes me 7 hours to finish running.

**Comment by yqy6611** [comment]
I think you can try genomad instead.
